R/chunk.R

```diff
@@ -0,0 +1,23 @@
+#' Register Chunk Callback
+#'
+#' Registers a callback function to be executed after a chunk within an R Markdown document is run
+#'
+#' @param callback A callback function that returns a list of html output to be
+#' displayed after a chunk is executed. The callback will be passed the parameter chunkName and chunkCode.
+#' @return A handle that can be used to unregister the chunk.
+#' @export
```

Comment:
It could use `@seealso unregisterChunkCallback` to document for the user explicitly (with a link) the companion function.

Reply:
Thanks Kevin! I made the update. Didn't add any additional text to the link because I think it's pretty self explanatory.